packages/desktop-client/src/components/reports/NetWorth.js

```diff
@@ -66,6 +69,16 @@ function NetWorth({ accounts }) {
     setEnd(end);
   }
 
+  const onApplyFilter = newFilter => {
+    setFilters(state => [...state, newFilter]);
+  };
+  const onUpdateFilter = (oldFilter, updatedFilter) => {
+    setFilters(state => state.map(f => (f === oldFilter ? updatedFilter : f)));
+  };
+  const onDeleteFilter = deletedFilter => {
+    setFilters(state => state.filter(f => f !== deletedFilter));
+  };
```

Comment:
Do you think this could be abstracted out? (Or moved into the filter element itself)?

Reply:
Makes sense: https://github.com/actualbudget/actual/pull/994/commits/23fb3d417963d3e0735cc050195c5368392b39c3

Once the `Account` components are moved from class-components to functions, then we can use it there too.